component/component/main.jsonnet

```diff
@@ -84,6 +84,7 @@ local ns = kube.Namespace(params.namespace) {
   metadata+: {
     labels+: {
       'openshift.io/cluster-monitoring': 'true',
+      'appuio.io/organization': 'vshn',
```

Comment:
What's the purpose of this?

Reply:
That is required for the VSHNMinio instance we deploy on the Lab. Without an organization label, the quota check will fail.